editor/src/components/canvas/controls/select-mode/absolute-resize-control.tsx

```diff
@@ -247,6 +259,46 @@ const ResizeEdge = React.memo(
   }),
 )
 
+interface SizeLabelProps {
+  targets: Array<ElementPath>
+}
+
+const SizeLabel = React.memo(
+  React.forwardRef<HTMLDivElement, SizeLabelProps>(({ targets }, ref) => {
+    const scale = useEditorState((store) => store.editor.canvas.scale, 'Resizelabel scale')
+    const colorTheme = useColorTheme()
+    const metadata = useEditorState((store) => getMetadata(store.editor), 'xxx')
+    const bbox = boundingRectangleArray(
+      targets.map((t) => MetadataUtils.getFrameInCanvasCoords(t, metadata)),
+    )
+    return (
+      <div
+        ref={ref}
+        style={{
+          position: 'absolute',
+          pointerEvents: 'none',
+          display: 'flex',
+          justifyContent: 'center',
+        }}
+        data-testid='parent-resize-label'
+      >
+        <div
+          style={{
+            marginTop: 8 / scale,
+            padding: 4 / scale,
+            borderRadius: 4 / scale,
+            color: colorTheme.white.value,
+            backgroundColor: colorTheme.secondaryBlue.value,
+            fontSize: 12 / scale,
+          }}
+        >
+          {bbox != null && `${bbox.width} x ${bbox.height}`}
```

Comment:
Truthiness alert.

Reply:
as far as I can see `bbox` is `CanvasRectangle | null` (since it comes from `boundingRectangleArray`, null is returned when the function is called with 0 rectangles, but in this case the control isn't rendered anyway)

Reply:
oh I already added this in [b09a3c2](https://github.com/concrete-utopia/utopia/pull/2962/commits/b09a3c2bb070f1d7c35c721892e32c08651e4150) (I got the spirit of your original comment)